resuscitation.csl

```diff
@@ -0,0 +1,271 @@
+      </if>
+      <else-if type="entry-encyclopedia" variable="author">
+        <text macro="editor-container"/>
+      </else-if>
+      <else-if type="chapter">
+        <text macro="editor-container"/>
+      </else-if>
+      <else-if type="patent" match="any">
+        <text variable="authority" suffix=" "/>
+        <text variable="number" suffix=","/>
+      </else-if>
+      <else>
+        <text variable="container-title" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <group delimiter=" ">
+      <choose>
+        <if is-numeric="edition">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </group>
+  </macro>
+  <macro name="city">
+    <text variable="publisher-place"/>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher"/>
+  </macro>
+  <macro name="year">
+    <choose>
+      <if type="legislation bill patent" match="any">
+        <date form="text" variable="issued">
+          <date-part name="year"/>
+          <date-part name="month" form="short"/>
+          <date-part name="day"/>
+        </date>
+      </if>
+      <else-if type="article-newspaper broadcast" match="any">
+        <date variable="issued" delimiter=" ">
+          <date-part name="year"/>
+          <date-part name="month" form="short"/>
+          <date-part name="day"/>
+        </date>
+      </else-if>
+      <else-if type="chapter article-journal" match="any">
+        <date variable="issued" suffix=";">
+          <date-part name="year"/>
+        </date>
+      </else-if>
+      <else>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </else>
+    </choose>
+  </macro>
+  <macro name="accessed">
+    <choose>
+      <if type="legislation bill" match="any"/>
+      <else-if variable="URL">
+        <group delimiter=" " prefix="[" suffix="].">
+          <text term="accessed"/>
+          <date delimiter=" " variable="accessed">
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="entry">
+    <choose>
+      <if type="entry-encyclopedia">
+        <choose>
+          <if variable="author"/>
+          <else>
+            <text variable="title"/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="post-weblog">
+        <text variable="title" suffix=";"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="locator">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <group delimiter=":">
+          <group>
+            <text variable="volume"/>
+            <group delimiter=" " prefix="(" suffix=")">
+              <date form="text" variable="issued">
+                <date-part name="month"/>
```

Comment:
@POBrien333, if you use a predefined date format with `form`, the `<date-part/>` elements only affect the formatting of predefined date parts, not whether they show or not. See http://docs.citationstyles.org/en/stable/specification.html#date

You can use `<date form="text" variable="issued" date-parts="year">`, or, and this is probably simpler, just delete `form="text"`.